HomeWork/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System.Numerics;
+using System.Security.Cryptography;
+
+Console.WriteLine("Enter the number of array elements");
+string size = Console.ReadLine();
+
+Console.WriteLine();
+
+int[] elements = new int[Convert.ToInt32(size)];
+
+
+
+for (int i = 0; i < Convert.ToInt32(size); i++)
+{
+    elements[i] = new Random().Next(1, 26);
+}
+
+
+int[] even = new int[Convert.ToInt32(size)];
+int[] uneven = new int[Convert.ToInt32(size)]; 
+
+
+for (int i = 0; i < Convert.ToInt32(size); i++)
```

Comment:
We have converted this before, it's better to put this value to a local variable